### DISCUSSION TITLE
Can we separate pan and zoom sensitivity ?

### DISCUSSION THREAD

**Discussion by nicolas-daures** [google/model-viewer#4401] [Ideas]
### Description

When the distance between the minimum radius and the maximum radius is big, the mousewheel zooms too much. Exposing the zoom-sensitivity and pan-sensitivity properties would allow for better control.

#### Live Demo
https://glitch.com/edit/#!/satin-deeply-hound

### Version
- model-viewer: v3.1.1

### Browser Affected
- [x] Chrome, version: 115.0.5790.110
- [x] Edge
- [x] Firefox
- [ ] IE
- [ ] Safari

### OS
- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

**Comment by elalish** [comment]
In addition to `orbit-sensitivity`, we also have an overall https://modelviewer.dev/docs/#entrydocs-stagingandcameras-properties-inputSensitivity. If that isn't enough, we'd be open to a PR to add more control.

**Reply by nicolas-daures** [reply]
Thank you for your reply.

The problem with `input-sensitivity` is that it impacts both zoom and pan. So if I lower the `input-sensitivity` so that the zoom is correct, the pan will be slow.

**Reply by elalish** [reply]
Fair enough - would you like to make a PR?

**Reply by nicolas-daures** [reply]
Here is the PR: https://github.com/google/model-viewer/pull/4412
